src/application/hooks/useInventory.js

```diff
@@ -0,0 +1,20 @@
+import { useContext } from "react";
+import { InventoryContext } from "../providers/InventoryProvider";
+
+export const useInventory = () => {
+  const value = useContext(InventoryContext);
+
+  if (value === undefined) {
+    throw new Error("You forgot InventoryProvider!");
+  }
+
+  const { inventory } = value;
+
+
+  const isLoading = inventory === undefined;
```

Comment:
```suggestion
  const { inventory } = value;
  
  const isLoading = inventory === undefined;
```

Evite pular duas linhas assim no meio do código, uma é suficiente se for necessário!